queue/console/queue_failed_command.go

```diff
@@ -0,0 +1,88 @@
+package console
+
+import (
+	"github.com/goravel/framework/contracts/config"
+	"github.com/goravel/framework/contracts/console"
+	"github.com/goravel/framework/contracts/console/command"
+	contractsdb "github.com/goravel/framework/contracts/database/db"
+	contractsqueue "github.com/goravel/framework/contracts/queue"
+	"github.com/goravel/framework/errors"
+	"github.com/goravel/framework/support/carbon"
+	"github.com/goravel/framework/support/color"
+)
+
+type QueueFailedCommand struct {
+	failedJobQuery contractsdb.Query
+	queue          contractsqueue.Queue
+}
+
+func NewQueueFailedCommand(config config.Config, db contractsdb.DB, queue contractsqueue.Queue) *QueueFailedCommand {
+	failedDatabase := config.GetString("queue.failed.database")
+	failedTable := config.GetString("queue.failed.table")
+
+	var failedJobQuery contractsdb.Query
+	if db != nil {
+		failedJobQuery = db.Connection(failedDatabase).Table(failedTable)
+	}
```

Comment:
The database may not be ready when registering the command. It would be better to move this logic into the `Handle` method and attempt to get the database connection when the command is executed.